app/src/main/res/values/strings.xml

```diff
@@ -32,4 +32,9 @@
     <string name="newPostTitle">"New Post"</string>
     <string name="newPostDescriptionPlaceholder">" What's on your mind today?"</string>
     <string name="postButton">"Post"</string>
+    <string name="account">"Account"</string>
+    <string name="termsAndConditions">"Terms &amp; Conditions"</string>
+    <string name="privacyPolicy">"Privacy Policy"</string>
+    <string name="about">"About"</string>
+    <string name="logout">"Logout"</string>
```

Comment:
i see these aren't used yet, is it for your next PR

Reply:
Yes, it will be used when I create the list of setting optiosn